content/usgs-packages/dataRetrieval_Intro.md

```diff
@@ -0,0 +1,40 @@
+---
+author: Lindsay R. Carr
+date: 
+slug: dataRetrieval-intro
+title: dataRetrieval - Introduction
+image: img/main/intro-icons-300px/r-logo.png
+identifier: 
+menu:
+  main:
+    parent: Introduction to USGS R Packages
+    weight: 2
+draft: true
+---
+Lesson Summary
+--------------
+
+This lesson will focus on finding and retrieving hydrologic time series data using the USGS R package, `dataRetrieval`. The package was created to make querying and downloading hydrologic data from the web easier and less error-prone. The package allows users to easily access data stored in the USGS National Water Information System (NWIS) and the multi-agency database, Water Quality Portal (WQP). NWIS only contains data collected by or for the USGS. Conversely, WQP is a database that aggregates water quality data from multiple agencies, including USGS, Environmental Protection Agency (EPA), US Department of Agriculture (USDA), and many state, tribal, and local agencies.
+
+`dataRetrieval` functions take user-defined arguments and construct web service calls to the intended database. The web service returns the data as XML (a standard data structure), and `dataRetrieval` takes care of parsing that into a useable R data.frame, complete with metadata. When web services change, `dataRetrieval` users aren't affected because the package maintainers will update the functions to handle these modifications. This is what makes `dataRetrieval` so user-friendly.
```

Comment:
I'd scratch "to the intended database". They are web services calls, I don't want to confuse people in thinking that we are directly querying a database.